tests/e2e/upgrade.go

```diff
@@ -0,0 +1,169 @@
+package e2e
+
+import (
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pingcap/tidb-operator/pkg/apis/pingcap.com/v1alpha1"
+	"github.com/pingcap/tidb-operator/pkg/controller"
+	apps "k8s.io/api/apps/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	upgradeVersion = "v2.0.0"
```

Comment:
use v2.0.5 instead, downgrade may be unsafe

Reply:
the v2.0.5 version is not exist